### PR TITLE
Error handling km

### LINKED ISSUE #77: Update 'Empty Photo List' asset
Currently using the empty guest list asset in `PhotoListForm` as a placeholder - can you make one that says the photo list is empty?

### LINKED ISSUE #81: Error handling: Can't continue from `GuestList` view to `PhotoListForm` view (via `Add Photos` button) if guest list is empty

### LINKED ISSUE #113: Add image assets to dashboard sorting
Add image assets for empty wedding results:

- [ ] No Upcoming Weddings
- [ ] No Archived Weddings
- [ ] No Weddings Today

### LINKED ISSUE #114: Add "finished shooting" asset to end of carousel
Indicate the end of shooting for the photo carousel with an image or text

### LINKED ISSUE #117: Change favico and tab title

### LINKED ISSUE #119: Add default image from front end

### PR DESCRIPTION
**Pull request by Kristiannmiller**
# Pull Request

## What's this PR do? 

- [X] New feature (non-breaking change which adds functionality)
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] Testing
- [X] Styling
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Description of changes?
- Adds error handling image assets
- Adds empty photo list image to render
- Adds empty wedding sorting images to render
- Adds isLoading hooks to state in PhotoList and Vendor-Dashboard
- Adds loading animation to VendorDashboard and PhotoList components
- Adds alert to forward button from GuestList component to prevent user from creating a photo list without guests in place
- Adds start and end image assets to PhotoShoot carousel
- Adds branded favicon and correct title to tab
- Adds conditional for default image if no image link is given in AddWeddingForm

## What are the relevant tickets? 
* Addresses
* Closes #119 
* Closes #117 
* Closes #114 
* Closes #81 
* Closes #77 
* Closes #113

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -25,7 +25,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>MatriMania</title>
```
Comment (ericberglund117) [inline public/index.html:28]: Nice Catch

File: src/components/GuestList/GuestList.tsx [hunk 2]
```diff
@@ -100,6 +100,14 @@ const GuestList: React.FC<WeddingData> = ({
     setGuests(filteredGuestList)
   }
 
+  const changeToPhotoList = () => {
+    if (guests.length === 0){
+      alert("Please add at least one guest before moving on to photos")
+    } else {
+      changeView('editPhotoListView')
+    }
+  }
```
Comment (ericberglund117) [inline src/components/GuestList/GuestList.tsx:109]: I like this error handling!

**Review by ericberglund117 — APPROVED** [review]
All of this looks great!

File: src/components/PhotoShootView/PhotoShootView.tsx [hunk 3]
```diff
@@ -57,7 +59,21 @@ const PhotoShootView: React.FC<PhotoShootData> = ({
...
+			return (
+				<StyledCard contents="shoot">
+					<img src={start} className="carouselImg" alt="click the next button to get started"/>
+				</StyledCard>
+			)
+		}
+		if(carousel > photoList.length) {
+			return (
+				<StyledCard contents="shoot">
+					<img src={done} className="carouselImg" alt="Good work! You're all done!"/>
```
Comment (hannakim91) [inline src/components/PhotoShootView/PhotoShootView.tsx:72]: Nice touch to signify the job is done!

**Review by hannakim91 — APPROVED** [review]

File: src/components/AddWeddingForm/AddWeddingForm.tsx [hunk 4]
```diff
@@ -29,12 +29,22 @@ const AddWeddingForm: React.FC<Props> = ({
...
-      name,
-      email,
-      date: weddingDate,
-      image
+    if(image === '') {
+      newWedding = {
+        name,
+        email,
+        date: weddingDate,
+        image: 'https://user-images.githubusercontent.com/65047537/106202246-dc00b080-6176-11eb-8067-5c7798af9a1b.jpg'
```
[the remark this answers is not in the document]
Comment (Kristiannmiller) [reply src/components/AddWeddingForm/AddWeddingForm.tsx:39]: Love it! Good call!

**Pull request merged**